### PR TITLE
Security consideration for MPS

### LINKED ISSUE #249: Improve Maximum Packet Size Considerations text
Olivier's comment in https://mailarchive.ietf.org/arch/msg/tsvwg/T-9P8iVoU5i89VK9QJXnOG_IsL4/

```
If MPDCCP uses the minimum MPS accross all paths, there is a risk that 
an on-path attack on one path forces the utilization of very small 
packets on all paths. MPDCCP should be able to detect this attack and 
drop paths with a too low MPS
```

### PR DESCRIPTION
**Pull request by markusa**
Closes #249

### CODE REVIEW

File: draft-ietf-tsvwg-multipath-dccp.mkd [hunk 1]
```diff
@@ -1595,6 +1595,13 @@ recipient to target other hosts in the network.  Therefore,
 implementations should consider heuristics at both the
 sender and receiver to reduce the impact of this.
 
+As described in {{mps}}, a Maximum Packet Size (MPS) is maintained for a MP-DCCP connection.
+If MP-DCCP uses a minimum MPS across all paths,
```
Comment (abrunstrom) [inline draft-ietf-tsvwg-multipath-dccp.mkd:1599]: If -- As

It is specified in the MPS section that MP-DCCP uses a minimum MPS accross all paths, and I do not see how using a per path value would work, so "if" does not seem appropriate.

**Review by abrunstrom — APPROVED** [review]
Added a nit, otherwise ready.

File: draft-ietf-tsvwg-multipath-dccp.mkd (hunk 1 above)
Comment (markusa) [reply draft-ietf-tsvwg-multipath-dccp.mkd:1599]: I'm not sure if this rather could be interpreted as a recommendation. Copied from the section:
`The DCCP application interface SHOULD allow the application to discover the current MPS. This reflects the current supported largest size for the data stream that can be used across the set of all active MP-DCCP subflows.`

We use SHOULD to enable the feature of minimum MPS, which is not a MUST and therefore up to the implementer. At least that is my interpretation.

What do you think, @abrunstrom?

Comment (markusa) [reply draft-ietf-tsvwg-multipath-dccp.mkd]: Thank you. I adopted your suggestion in https://github.com/tsvwg/ietf-multipath-dccp/pull/272/commits/14e95972c2b158d2930ade3991dd5bb51d00b782

**Pull request merged**